lib/build-components/build_components/templates/python/macros.j2

```diff
@@ -170,9 +170,6 @@ RUN groupadd -f {{user_info.group}} \
  && rm -rf /var/lib/apt/lists/* \
  && echo >> /etc/sudoers "{{user_info.name}} ALL=(ALL) NOPASSWD: ALL"
 {% endif %}
-
```

Comment:
So I see only removing the USER command. Do we add it somewhere later in the file?

Reply:
oops. it's in the other file. I'll read more closely.